chromedriver-bin/update.sh

```diff
@@ -1,8 +1,15 @@
 #!/usr/bin/env bash
 # Author: Kévin Dunglas <dunglas@gmail.com>
-# Download the last version of ChromeDriver binaries
+# Download the last compatible version of ChromeDriver binaries, falling back to last version
 
-latest=$(curl -s https://chromedriver.storage.googleapis.com/LATEST_RELEASE)
+current=$(chromium --product-version)
```

Comment:
`chromium` may not exist if the real Chrome is used or if the system isn't Linux. For instance I use Chrome on Mac OS and this breaks the script on my computer.